pkgs/data/fonts/kacst/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation {
+  pname = "kacst";
+  version = "2.01";
+
+  src = fetchurl {
+    url = "mirror://debian/pool/main/f/fonts-kacst/fonts-kacst_2.01+mry.orig.tar.bz2";
```

(suggestion, non-blocking)

A common convention is to replace version tokens with the `${version}` variable.

```suggestion
    url = "mirror://debian/pool/main/f/fonts-kacst/fonts-kacst_${version}+mry.orig.tar.bz2";
```

---

I'll investigate the failed check